### ISSUE TITLE
Multi-tenant

### ISSUE THREAD

**Issue by alexometis** [qlik-oss/qlik-snowflake-monitor#11]
Many companies have multiple Snowflake Tenants. It would be great to see functionality to support this added to the app.

**Comment by badgerriaan** [comment]
Yes, we use 2 main Snowflake accounts, but you can easily have multiple Accounts (E.g. DEV, QA, LIVE or country-based accounts) under your organisation under Snowflake. If feasible, it would be nice to analyze more than one in a single app.
